cg/meta/upload/scoutapi.py

```diff
@@ -74,13 +79,23 @@ def build_samples(self, analysis_obj: models.Analysis, hk_version_id: int = None
                 "tissue_type": lims_sample.get("source", "unknown"),
                 "vcf2cytosure": vcf2cytosure_path,
             }
-            yield sample
+            scout_sample = ScoutIndividual(**sample)
+            yield scout_sample.dict(exclude_none=True)
 
     def generate_config(self, analysis_obj: models.Analysis, rank_score_threshold: int = 5) -> dict:
         """Fetch data about an analysis to load Scout."""
-        analysis_date = analysis_obj.started_at or analysis_obj.completed_at
-        hk_version = self.housekeeper.version(analysis_obj.family.internal_id, analysis_date)
-        analysis_data = self.analysis.get_latest_metadata(analysis_obj.family.internal_id)
+        LOG.info("Generate scout load config")
+        analysis_date: datetime = analysis_obj.started_at or analysis_obj.completed_at
+        LOG.debug("Found analysis date %s", analysis_date)
+        hk_version: hk_models.Version = self.housekeeper.version(
+            analysis_obj.family.internal_id, analysis_date
+        )
+        LOG.debug("Found housekeeper version %s", hk_version.id)
+        analysis_data: dict = self.analysis.get_latest_metadata(analysis_obj.family.internal_id)
+        raw_genome_build = analysis_data.get("genome_build", "37")
+        genome_build = "37"
+        if "38" in raw_genome_build:
+            genome_build = "38"
```

Comment:
```suggestion
        raw_genome_build = analysis_data.get("genome_build", "37")
        genome_build = "37"
        if "38" in raw_genome_build:
            genome_build = "38"
```

raw_genome_build = analysis_data.get("genome_build")
genome_build = "38" if "38" in raw_genome_build else "37"

Reply:
or even shorter

Reply:
I think that will fail if "genome_build" does not exist. But I get the point